src/kv/store.h

```diff
@@ -746,6 +746,17 @@ namespace kv
         {
           success = DeserialiseSuccess::PASS_NEW_VIEW;
         }
+        else if (views.find("ccf.signatures") != views.end())
+        {
```

Comment:
Just for my understanding, is it right that this the branch we take when re-execute on a secondary? What we're doing here is taking the primary's signature, and applying it without re-execution?